google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -652,7 +652,11 @@ public void close() throws Exception {
       return;
     }
     closed = true;
-    rpc.close();
+    try {
+      flush();
```

Comment:
Won't this potentially block and prevent the rpc connection from closing. This is demonstrated by the test - it's blocking until flush succeeds.

Reply:
@chingor13, what should the behavior be for `close()` without a previous `flush()`?  In Cloud Bigtable's custom client, we always flush first, which I think meets the expectation of the user.   However, I do understand that there isn't a manual flush for Logging, which changes the dynamics; IIUC, Logging will wait for some time induced flushing.

@igorbernstein2 is working on manual flush() mechanism so that Cloud Bigtable's use of flush works as expected; hopefully, that will allow a fix for other implementations, like logging.